source/html/js/app/statemachine.js

```diff
@@ -20,16 +20,16 @@ const configurationStateMachine = new machina.Fsm({
         },
         "get-saved-connection": {
             _onEnter: function () {
-                var current_connection = connections.get_current();
+                let current_connection = connections.get_current();
```

Comment:
Same const as above

Reply:
there were tons of places where it uses `let` where it could have been `const`.
It felt awkward to change only in the few places you mentioned.. so... I have changed it everywhere in the project.

In order to, hopefully, make things easier when reviewing, I put the `let` -> `const` changes into its own commit.